api/plugins/tests/integration/controlplane/control_plane.go

```diff
@@ -77,6 +78,15 @@ func NewControlPlane() *ControlPlane {
 	return cp
 }
 
+func getRandomString(n int) string {
+	var Letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
```

Comment:
I have two questions here.
1. Ascii is enough here. Do we need `rune` here?
2. could change `Letters` to `letters`?
3. If `rune` is unnecessary, could we declare `Letters` as a constant?

Reply:
@zll600 
Applied. Thanks!